visidata/guides/CommandsSheet.md

```diff
@@ -0,0 +1,28 @@
+# How to find commands
+
+## Use the fuzzy finder to search by name
+
+- {help.commands.exec_longname}
+
+Start typing a command longname or keyword in its helpstring.
```

Comment:
"Fuzzy match command by longname and help string."

Header maybe should be "## Command palette"?  That's what we called it internally when developing it.